lib/internal/child_process.js

```diff
@@ -713,9 +712,8 @@ function setupChannel(target, channel, serializationMode) {
     } else if (typeof options === 'function') {
       callback = options;
       options = undefined;
-    } else if (options !== undefined &&
-               (options === null || typeof options !== 'object')) {
-      throw new ERR_INVALID_ARG_TYPE('options', 'Object', options);
+    } else if (options !== undefined) {
+      validateObject(options, 'options');
```

Comment:
FYI, this change broke `cheerio` through the `workerpool` module: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/2604/nodes=fedora-last-latest-x64/testReport/junit/(root)/citgm/cheerio_v1_0_0_rc_5/

See https://github.com/josdejong/workerpool/blob/b0e72695fc26df60e224c2d2eafc402f3097a6f4/src/WorkerHandler.js#L262

It would probably be nice to open a PR to fix that module. It seems to have been working by accident until now.

Reply:
Done in https://github.com/josdejong/workerpool/pull/230.

Reply:
We should probably also do <https://github.com/nodejs/node/pull/37047>.